### ISSUE TITLE
Create Endpoints to handle Management Command Jobs

### ISSUE THREAD

**Issue by italogsfernandes** [geosolutions-it/geonode#950]
Refer Epic #909:
Issue refers to a proposal:
> "to implement a mechanism inside GeoNode to generate an HTTP API for any management command. The idea is to expose management over a common HTTP endpoint, granting of course full control of which commands we want to be exposed and under which security restrictions."

## Proposed Solution
In the issue #949, an app was created, not it needs to provide some endpoints and features to manage the Jobs.
* Create an ModelViewSet for ManagementCommandJob.
    * [x] `GET` `/api/v2/management/jobs/`:  **List** jobs, allow filters by `status`, `app_name`, `command_name`.
    * [x] `POST` `/api/v2/management/jobs/{id}`: **Details** of a job.
    * [x] `PATCH` `/api/v2/management/jobs/{id}`: **Update** the status of a job.
        * Note: finished jobs can not be changed.
        * [x] A queued and running job can be stopped --> **stop/cancel** action.
        * [x] A created job can be queued --> **start** action.
*  [x] Add new endpoints to the Management `ApiView`.
    * [x]  `GET` `/api/v2/management/<cmd_name>/jobs` : **List** jobs filtered by `cmd_name`.
* [x] Add the **start** and **stop** options to the django admin actions.
* [x] Add a **create** option in django admin, the available commands should be displayed as a ChoicesField.
* [x] Tests and validation

##  Alternative solutions considered.
See closed issues related to the epic #909.

## Additional context
None

**Comment by giohappy** [comment]
@italogsfernandes I will close this once the port to master is done

**Comment by giohappy** [comment]
closing in favour of https://github.com/geonode/geonode/issues/8158
